components/scream/cime_config/buildnml

```diff
@@ -3,9 +3,11 @@
 """
 Namelist creator for E3SM's SCREAM component
 
-This scripts takes the namelist_defaults_scream.xml file from the repo
+This script takes the namelist_defaults_scream.xml file from the repo
 and produces the processed $case/namelist_scream.xml. From the processed
 file, raw input files are produced which should not be modified by users.
+The user is allowed to modify $case/namelist_scream.xml either directly
```

Comment:
Do we really want to let them modify it directly? I mean, we can't prevent this from happening, but we should *highly* discourage it. If the user accidentally modifies the structure of the file, we might generate something completely off, with an invalid xml format being our best-case scenario.

Reply:
Remember, with the changes introduced yesterday to the prototype, the metadata of parameters, computed from the repo XML file, must be obeyed in the $case file. So it's not completely left up to the user to not make a mistake.